src/common/utils/random-bytes.injectable.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { randomBytes } from "crypto";
+import { promisify } from "util";
+
+export type RandomBytes = (size: number) => Promise<Buffer>;
+
+const randomBytesInjectable = getInjectable({
+  id: "random-bytes",
+  instantiate: (): RandomBytes => promisify(randomBytes),
```

Comment:
Is there a reason for this to be promisified? Could be just sync?

Reply:
No because the interface of `randomBytes` returns its result in a callback.

Reply:
The reason why we don't want the sync version is because it will block the entire application until enough entropy is present.